src/components/Footer.js

```diff
@@ -29,14 +29,14 @@ export default class Footer extends Component {
     const { socials } = this.props
     return (
       <footer className="Footer">
-        <Col sm={4} className="Footer-element">
-          <SocialIcons className="Footer-socials" entity={socials} />
+        <Col sm={4} xs={4}>
+          <SocialIcons className="socials" entity={socials} />
         </Col>
-        <Col sm={4} className="Footer-element Footer-center">
+        <Col xs={4} className="text-center">
           Are you ClearlyDefined?
         </Col>
-        <Col sm={4} className="Footer-element">
-          <div className="pull-right Footer-right">
+        <Col sm={4} xs={4} xsOffset={1}>
+          <div className="pull-right right">
             <a href="https://docs.clearlydefined.io/legal/terms" target="_blank" rel="noopener noreferrer">
```

Comment:
can we force these to not wrap?

original:
![image](https://user-images.githubusercontent.com/5168796/53515548-1808d380-3a7f-11e9-818c-2ed37869ddfd.png)

proposed:
![image](https://user-images.githubusercontent.com/5168796/53515538-0f180200-3a7f-11e9-8efc-5a14f89f57c4.png)

Reply:
looks good. Should I hide the "pipe" characters "I" only for mobile?